index.html

```diff
@@ -72,45 +71,46 @@ <h2>Facts</h2>
                 </div>
                 <div class="gridM flex grid">
                 <div class="child flex">
-                    <img src="./media/arquitecto.png" width="75px" height="50px" alt="placeholder">
+                    <img src="./media/arquitecto.png" width="75" height="50" alt="placeholder">
                     <h3>Presentation</h3>
                     <p>Better presentations at events</p>
                 </div>
                 <div class="child flex">
-                    <img src="./media/arquitectura.png" width="75px" height="50px" alt="placeholder">
+                    <img src="./media/arquitectura.png" width="75" height="50" alt="placeholder">
                     <h3>Structuring</h3>
                     <p>Better project elaborations</p>
                 </div>
                 <div class="child flex">
-                    <img src="./media/arquitecto.png" width="75px" height="50px" alt="placeholder">
+                    <img src="./media/arquitecto.png" width="75" height="50" alt="placeholder">
                     <h3>Best Price</h3>
                     <p>Most paid structure and plane</p>
                 </div>
                 <div class="child flex">
-                    <img src="./media/desing.png" width="95px" height="50px" alt="placeholder">
+                    <img src="./media/desing.png" width="95" height="50" alt="placeholder">
                     <h3>layout</h3>
                     <p>Best layout, design and <br> presentation at conference </p>
                 </div>
                 <div class="child flex">
-                    <img src="./media/idea.png" width="95px" height="50px" alt="placeholder">
+                    <img src="./media/idea.png" width="95" height="50" alt="placeholder">
                     <h3>Best idea</h3>
                     <p>most creative idea at <br> presentation</p>
                 </div>
                 <div class="child flex">
-                    <img src="./media/desing.png" width="95px" height="50px" alt="placeholder">
+                    <img src="./media/desing.png" width="95" height="50" alt="placeholder">
                     <h3>Best Design</h3>
                     <p>most expensive and <br>  sold design</p>
                 </div>
-
-                <BUtton>More info?</BUtton>
+                <a class="more-info" href="about.html">More Info </a>
 
                 <div class="more_info flex">
-                    <a href="#"> See more information</a>
+                    <a href="https://www.pritzkerprize.com/"> See more information</a>
                 </div>
             </div>
             </section>
+            <h2 class="featured">Featured Speakers</h2>
 
             <section id="architect" class="grid_Winer">
```

Comment:
- [ ] As stated above,  Please follow the design provided to implement some features, Compare the two images again:

--------------------------------------Original-----------------------------------------------
![nav3](https://user-images.githubusercontent.com/55635977/128220317-a0df3557-c571-43c1-9431-255f68b37ff3.png)

--------------------------------------Yours-----------------------------------------------
![nav4](https://user-images.githubusercontent.com/55635977/128220233-e8a7b798-ee7c-4f60-9c3e-cebb571e6055.png)

Reply:
Sorry but what is this? 

Did you saw my video? my project works fine...

Reply:
This issue is not properly resolved 😿

Reply:
![image](https://user-images.githubusercontent.com/74747182/128398662-3321cee8-95e4-4d6c-9db0-01acec93c15a.png) 
Ammmm